worksheet_D/oxo.py

```diff
@@ -1,26 +1,64 @@
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.gameboard = []
+        for i in xrange(9):
+            self.gameboard.append(0)
+
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+        square = x + y * 3
```

Comment:
This calculation is repeated a few times -- might be good to wrap it in a method